dbsync/common/src/main/java/com/google/edwmigration/dbsync/common/ChecksumGenerator.java

```diff
@@ -48,7 +48,8 @@ public void generate(ChecksumConsumer<IOException> out, ByteSource in) throws IO
         // Yes, this masks the instance variable.
         int blockSize = rollingChecksum.getBlockSize();
         if (DEBUG) {
```

Comment:
Do we really need this flag? Looks like we can remove it from the code and just use logging with levels